### PR TITLE
routines navbar tab displaying all of the users routines

### PR DESCRIPTION
**Pull request by marywebby**

### CODE REVIEW

**Comment by marywebby** [comment]
resolves #15

**Comment by marywebby** [comment]
resolves #16

File: app/controllers/routines_controller.rb
Comment (Dylanm22) [inline app/controllers/routines_controller.rb:1]: Nice logic in the controller! code looks clean!

**Review by Dylanm22 — COMMENTED** [review]
this pull looks good!

**Review by Dylanm22 — APPROVED** [review]

**Pull request merged**